.storybook/main.js

```diff
@@ -1,79 +1,19 @@
+const webpackConfigDev = require('../webpack.dev');
 const path = require('path');
 
-const babelRules = config => {
-  config.module.rules.push({
-    test: /\.(ts|tsx)$/,
-    use: [
-      {
-        loader: require.resolve('babel-loader'),
-        options: {
-          envName: 'build',
-        },
-      }
-    ]
-  });
-  config.module.rules.push({
-    test: /\.(ts|tsx)$/,
-    use: [
-      {
-        loader: require.resolve('react-docgen-typescript-loader'),
-        options: {
-          shouldExtractLiteralValuesFromEnum: true,
-          tsconfigPath:  path.resolve('tsconfig.json')
-        },
-      },
-    ]
-  });
-  config.resolve.extensions.push('.ts', '.tsx');
-  return config;
-};
-
-const scssRules = {
-  test: /\.scss$/,
-  use: [
-    'style-loader',
-    {
-      loader: 'css-loader',
-      options: {
-        modules: true,
-      },
-    },
-    // 'postcss-loader', @TODO -- Add postcss config and re-enable loader.
-    'sass-loader',
-    {
-      loader: 'sass-resources-loader',
-      options: {
-        // Provide path to the file with resources
-        resources: [
-          './src/styles/variables/index.scss',
-          './src/styles/globals/utilities.scss',
-          './src/styles/reset.scss',
-        ],
-      },
-    },
-  ],
-};
-
-const fileLoaderRules = {
-  test: /\.(png|jpe?g|gif)$/i,
-  use: [{
-    loader: 'file-loader',
-  }, ],
-};
-
-const svgRules = config => {
-  const assetRule = config.module.rules.find(({ test }) => test.test(".svg"));
-  const assetLoader = {
-    loader: assetRule.loader,
-    options: assetRule.options || assetRule.query
-  };
-  // Merge our rule with existing assetLoader rules
-  config.module.rules.unshift({
-    test: /\.svg$/,
-    use: ["@svgr/webpack", assetLoader]
-  });
-  return config;
-}
+// const svgRules = config => {
```

Comment:
There's a few instances of commented code in the changes. Are we keeping those around for a reason?

Reply:
@mtblanton I was wanting to keep this config as I think we we will be needing it soon(ish) but I much prefer to delete it and start clean once we get to that point where we need to process svgs in our builds.